core/src/main/java/org/apache/accumulo/core/conf/PropertyType.java

```diff
@@ -136,6 +141,9 @@ public enum PropertyType {
       "An arbitrary string of characters whose format is unspecified and"
           + " interpreted based on the context of the property to which it applies."),
 
+  JSON("json", x -> new ValidJson().test(x),
```

Comment:
Would this create a new `ValidJson` object each time this is used? If so, is there a way to create a single instance for reuse?

Reply:
The test is fairly light weight, so I don't know how we could evaluate if it made a difference.  With 7fa8acabb7, the ObjectMapper is shared, so there should be even less impact on object creation.

Reply:
It doesn't make sense to make the object mapper static, and instantiate the ValidJson object each time in order to call it's test method. It's already a Predicate, so you don't need the `x -> ... .test(x)` stuff, you can just do:

```suggestion
  JSON("json", ValidJson::new,
```

Then, you don't have to worry about making the object mapper a static singleton... the PropertyType.JSON enum will just re-use the singleton `ValidJson::new` instance passed in for its predicate every time.

Reply:
It should be just `new ValidJson()`, my mistake.

Reply:
fixed in 33bc47d2a1